site/app/models/GradeableComponent.php

```diff
@@ -191,11 +191,15 @@ public function getGradedTAComments($nl, $show_students, $use_ascii = true) {
         foreach ($this->marks as $mark) {
             $points_string = "    ";
             if ($mark->getPoints() != 0) {
-              $points_string = sprintf("%4.1f",$mark->getPoints());
+              $roundPoints = round($mark->getPoints()/$gradeable->getPointPrecision())*$gradeable->getPointPrecision();
+              if ($gradeable->getPointPrecision() === 0.01)
```

Comment:
Instead of checking for equality with 0.01, this should be generalized for any precision into the hundredths, e.g., 0.05.
Also, instead of handling only tenths and hundredths, this should handle round number precision (= 1) and thousandths (= 0.001), etc.